v1/config.go

```diff
@@ -64,6 +64,27 @@ type RootFS struct {
 	DiffIDs []Hash `json:"diff_ids"`
 }
 
+// HealthConfig holds configuration settings for the HEALTHCHECK feature.
+type HealthConfig struct {
+	// Test is the test to perform to check that the container is healthy.
+	// An empty slice means to inherit the default.
+	// The options are:
+	// {} : inherit healthcheck
+	// {"NONE"} : disable healthcheck
+	// {"CMD", args...} : exec arguments directly
+	// {"CMD-SHELL", command} : run command with system's default shell
+	Test []string `json:",omitempty"`
```

Comment:
nit: should these be lowercased like `json:"test,omitempty"`

Reply:
Ah, basically just to make it lowercase when marshalled, to match the way we marshal all the other values in this file